posts/month-in-mathlib-aug-2021.md

```diff
@@ -0,0 +1,74 @@
+link: ''
+slug: month-in-mathlib-aug-2021
+tags: ''
+title: This month in mathlib (Aug 2021)
+type: text
+---
+
+# This month in mathlib (Aug 2021)
+
+## Highlighted PRs
+
+* [PR #8652](https://github.com/leanprover-community/mathlib/pull/8652) :: chore(*): update lean to 3.32.1
+
+  The community fork of Lean made two new
+  [releases](https://github.com/leanprover-community/lean/blob/master/doc/changes.md#3321c-12-august-2021)
+  `3.32.0` and `3.32.1`.
+  This is part of the preparations for porting mathlib to Lean 4.
+
+* [PR #8281](https://github.com/leanprover-community/mathlib/pull/8281) :: feat(topology,geometry/manifold): continuous and smooth partition of unity
+
+  See the [companion blogpost](continuous-partitions-of-unity/) for details.
+
+* Radon-Nikodym and Lebesgue decomposition. The four PRs
+  [PR #8645](https://github.com/leanprover-community/mathlib/pull/8645)
+  [PR #8687](https://github.com/leanprover-community/mathlib/pull/8687)
+  [PR #8763](https://github.com/leanprover-community/mathlib/pull/8763)
+  [PR #8875](https://github.com/leanprover-community/mathlib/pull/8875)
+  together contribute
+  the Lebesgue decomposition for sigma-finite measures
+  and the Radon-Nikodym theorem.
+
+* [PR #7978](https://github.com/leanprover-community/mathlib/pull/7978) :: feat(measure_theory/interval_integral): strong version of FTC-2
+
+  This PR signals the start of a serious attack on Stokes' theorem,
+  which will unlock a general approach to complex analysis.
+
+* [PR #8692](https://github.com/leanprover-community/mathlib/pull/8692) :: feat(field_theory): finite fields exist
+
+  Most of this PR had been lying around for ages,
+  but it was finally put together in mathlib.
+  It shows the existence and uniqueness of finite fields.
+
+* Among several PRs from the [Dedekind project](https://github.com/lean-forward/class-number), the two most noteworthy are
+  - [PR #8530](https://github.com/leanprover-community/mathlib/pull/8530) :: feat(ring_theory): ideals in a Dedekind domain have unique factorization
+  - [PR #8626](https://github.com/leanprover-community/mathlib/pull/8626) :: feat(ring_theory): define the ideal class group
+
+* [PR #8377](https://github.com/leanprover-community/mathlib/pull/8377) :: feat(analysis/complex/upper_half_plane): new file
+
+  This PR defines the complex upper half plane, together with the $\mathrm{SL}_2$-action.
+  Upcoming PRs will characterize the fundamental domain of the $\mathrm{SL}_2(\mathbb{Z})$-action.
+
+## Other mathematical contributions
+
+The following PRs are ordered by the date that they were merged into mathlib.
+
+* [PR #8424](https://github.com/leanprover-community/mathlib/pull/8424) :: feat(analysis/complex): prove that complex functions are conformal if and only if the functions are holomorphic/antiholomorphic with nonvanishing differential
+* [PR #4885](https://github.com/leanprover-community/mathlib/pull/4885) :: feat(category_theory/adjunction): general adjoint functor theorem
```

Comment:
Shouldn't this move to the highlight section? I thought that category theory people think this is a very important theorem.

Reply:
@b-mehta @semorrison @adamtopaz  what do you think? Do you want to write two sentences about the PR, and move it to the highlights?

Reply:
I think it should certainly go in the highlights section. If @b-mehta has some plans to use this theorem in the near future, it would be nice to add a couple of sentences along these lines.

Reply:
I don't have any particular plans for this, nor do I have a strong opinion on where it should go!